server.js

```diff
@@ -49,6 +49,14 @@ var running_id = 1
 var running_match_id = 1
 var awaiting_match_room = null
 
+
+// Manages creating and joining a custom room.
```

Comment:
Not a fan of this comment style.
If you want to break out the required parameters of join_room_json, that's fine, but it should be like:
// join_room_json required params:
//     game_version - blah blah
//     room_id - blah blah
// etc. for all params
Also, i probably want the comment inside the function for now.

database is a global here and is for logging stuff to the database.